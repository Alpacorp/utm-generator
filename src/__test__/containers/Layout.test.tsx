import React from "react";
import { render } from "@testing-library/react";
import Layout from "../../containers/Layout";

describe("Layout", () => {
  test("renders Layout component to match with snapshot", () => {
    const { container } = render(<Layout />);
    expect(container).toMatchSnapshot();
  });
});
