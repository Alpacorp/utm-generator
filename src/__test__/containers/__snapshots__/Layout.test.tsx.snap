// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders Layout component to match with snapshot 1`] = `<div />`;
